### PR TITLE
Setup messaging handling

### PR DESCRIPTION
**Pull request by gmittal**
I think for now this is all that is required on the backend to handle messaging. The Firebase on the volunteer frontend will do the rest.

@ankitr PTAL.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -177,19 +148,17 @@ app.post('/communication/incoming/sms', function (req, res) {
...
+  var contactID = hash.encodeHex(Buffer(message.from).toString("hex"));
+  db.child("messages").child(contactID).push(message);
 
   res.status(200).end();
 });
 
 
+// Send a message
+app.post('/communication/send', function (req, res) {
+
```
Comment (ankitr) [inline app.js:160]: Is there supposed to be something here?

Comment (gmittal) [inline app.js:160]: Not supposed to be anything yet.

**Comment by ankitr** [comment]
lgtm

**Comment by gmittal** [comment]
Fixed #13 @ankitr PTAL again.

File: util/imgur.sh [hunk 2]
```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
```
Comment (ankitr) [inline util/imgur.sh:1]: I might clean this up someday.

**Comment by ankitr** [comment]
lgtm

**Pull request merged**